docs/articles/playbooks/Expensify-Chat-Playbook-for-Conferences.md

```diff
@@ -0,0 +1,114 @@
+
+- Communicate logistics and key information
+- Foster conference wide attendee networking
+- Organize conversations by topic and audience
+- Continue conversations long after the event itself 
+- Digitize attendee social interaction
+
+Sounds good? Great! In order to ensure your team, your speakers, and your attendees have the best experience possible, we’ve created a guide on how to use Expensify Chat at your event. 
+
+_Let’s get started!_
+
+
+## Support
+
+Connect with your dedicated account manager in any new.expensify.com #admins room. Your account manager is excited to brainstorm the best ways to make the most out of your event and work through any questions you have about the setup steps below. 
+
+
+## Step by step instructions for setting up your conference on Expensify Chat
+
+Based on our experience running conferences atop Expensify Chat, we recommend the following simple steps:
+
+### Step 1: Create your event Workspace in Expensify 
+
+To create your event workspace in Expensify:
+1. In new.expensify.com: “+” > “New workspace” 
+2. Name the workspace e.g. “ExpensiCon”
+
+### Step 2: Setup all necessary Expensify Chat rooms you want to feature at your event
+
+*Protip*: Your account manager can complete this step with you. Chat with them in #admins on new.expensify.com to coordinate!
+
+To create a new chat room:
+1. Go to new.expensify.com
+2. Go to “+” > New room  
+3. Name to room e.g. “#social”
+4. Select the workspace created at step 1
+5. Select “Public” visibility 
+6. “Create room” > Copy/Paste room URL for use later
+7. Repeat for each room
+
+For an easy-to-follow event, we recommend creating these chat rooms:
+
+- *#social* - This room will include all attendees, speakers, and members of your organizing team. You can use this room to discuss social events, happy hours, dinners, or encourage attendees to mingle, share photos and connect. 
+- *#announce* - This room will be used as your main announcement channel, and should only be used by organizers to announce schedule updates or anything important that your attendees need to know. Everyone in your policy will be invited to this channel, but chatting in here isn’t encouraged so to keep the noise to a minimum.
+- *Create an individual room for each session* - Attendees will be able to engage with the speaker/session leader and can ask questions about their content either before/during/after the session.
+- *Create a room with your Expensify account manager/s* - We can use this room to coordinate using Expensify Chat before, during, and after the event. 
+
+### Step 3: Add chat room QR codes to the applicable session slide deck
+
+Gather QR codes:
+1. Go to new.exensify.com 
```

Comment:
This one says `exensify` - opened a quick PR [here](https://github.com/Expensify/App/pull/22917). Directly assigned you @chiragsalian since you already have context.